2025-02-09 21:08:36,262:DEBUG:certbot._internal.main:certbot version: 3.1.0
2025-02-09 21:08:36,262:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2025-02-09 21:08:36,262:DEBUG:certbot._internal.main:Arguments: ['--standalone', '-d', 'Secretbay.me', '--non-interactive', '--agree-tos', '-m', 'your_email@example.com']
2025-02-09 21:08:36,263:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2025-02-09 21:08:36,279:DEBUG:certbot._internal.log:Root logging level set at 30
2025-02-09 21:08:36,281:DEBUG:certbot._internal.plugins.selection:Requested authenticator standalone and installer None
2025-02-09 21:08:36,281:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * standalone
Description: Runs an HTTP server locally which serves the necessary validation files under the /.well-known/acme-challenge/ request path. Suitable if there is no HTTP server already running. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='standalone', value='certbot._internal.plugins.standalone:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.standalone.Authenticator object at 0x791ae0af0b00>
Prep: True
2025-02-09 21:08:36,284:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.standalone.Authenticator object at 0x791ae0af0b00> and installer None
2025-02-09 21:08:36,284:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator standalone, Installer None
2025-02-09 21:08:36,348:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/2219103505', new_authzr_uri=None, terms_of_service=None), 5419752d9ebef24cc83ce73e327da5c0, Meta(creation_dt=datetime.datetime(2025, 2, 9, 20, 8, 31, tzinfo=datetime.timezone.utc), creation_host='abe4a56e3107', register_to_eff=None))>
2025-02-09 21:08:36,349:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2025-02-09 21:08:36,351:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2025-02-09 21:08:36,767:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 828
2025-02-09 21:08:36,768:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 09 Feb 2025 21:08:36 GMT
Content-Type: application/json
Content-Length: 828
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "aikC609dE6c": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "profiles": {
      "classic": "The same profile you're accustomed to"
    },
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.4-April-3-2024.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert"
}
2025-02-09 21:08:36,769:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for secretbay.me
2025-02-09 21:08:36,773:DEBUG:acme.client:Requesting fresh nonce
2025-02-09 21:08:36,773:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2025-02-09 21:08:36,912:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2025-02-09 21:08:36,912:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 09 Feb 2025 21:08:36 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 4v_-BYbeYKIA-mGRlnQfPP9lJnnd1QkpdgtomHtay3xfquxpiXo
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2025-02-09 21:08:36,913:DEBUG:acme.client:Storing nonce: 4v_-BYbeYKIA-mGRlnQfPP9lJnnd1QkpdgtomHtay3xfquxpiXo
2025-02-09 21:08:36,913:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "secretbay.me"\n    }\n  ]\n}'
2025-02-09 21:08:36,916:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjIxOTEwMzUwNSIsICJub25jZSI6ICI0dl8tQlliZVlLSUEtbUdSbG5RZlBQOWxKbm5kMVFrcGRndG9tSHRheTN4ZnF1eHBpWG8iLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "Cmzahbm2B-yGOpjazU3Nodx7EXDKIfRGgQvaYorrBdtABI-kl6_P1ROnc4ARnCoYKkf4mr0e14J7hLIs1FZl0iBSce-_QXvWnrwpx74TOs7u3BJLdxPA9jpFmcsM0DmSP96CYK7NB0g1rtdKch5bqlhydGVAVodpfD6kAAyQNzLImX3ys8k6qbGyVVI5Axq3pwrDsXGETdHNA2y2j5hcSwxAT4IBklkOGpnXUpEZpiDQbUXU5AHsCe_GKNReTYHi9DRyT0aDOqdIbAvjT1Oznj9vCcQsZshdav2M7vH_cRIiumyVj3BZ31KKszxsgbZoimmvkQHPBHiAyFp1qPyVsw",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogInNlY3JldGJheS5tZSIKICAgIH0KICBdCn0"
}
2025-02-09 21:08:37,233:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 346
2025-02-09 21:08:37,233:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Sun, 09 Feb 2025 21:08:37 GMT
Content-Type: application/json
Content-Length: 346
Connection: keep-alive
Boulder-Requester: 2219103505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/2219103505/352806282745
Replay-Nonce: 4v_-BYbegViTJkuIPyPgpwY8kxDxewvghQ0cfXNIVikH0XlgLQQ
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2025-02-16T21:08:37Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "secretbay.me"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz/2219103505/473521759415"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/2219103505/352806282745"
}
2025-02-09 21:08:37,234:DEBUG:acme.client:Storing nonce: 4v_-BYbegViTJkuIPyPgpwY8kxDxewvghQ0cfXNIVikH0XlgLQQ
2025-02-09 21:08:37,234:DEBUG:acme.client:JWS payload:
b''
2025-02-09 21:08:37,236:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2219103505/473521759415:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjIxOTEwMzUwNSIsICJub25jZSI6ICI0dl8tQlliZWdWaVRKa3VJUHlQZ3B3WThreER4ZXd2Z2hRMGNmWE5JVmlrSDBYbGdMUVEiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIyMTkxMDM1MDUvNDczNTIxNzU5NDE1In0",
  "signature": "Vl_Qzm_WEXDp47PuxP1COmSExfdXQTgEsBjSUv600AuXTjevSsBWZNgoi9QjEn8summGig_kx6mzo_6UePfUcYme1x_Eny1t3p5xAaoVCh70zQ1spGuuAqNKiuiLDLWGya_wOw5vXGHX0KuETDkZeBlEmFLLU_mWDYJdi5ZO789eyjyged6czrcXJQr7MOQ3njFVMBV8IWDLHm4iUWKQLKCbxKJT-DKFK_nUaRu7a5cIVSqyFwxLjqXZWSdnrWjomU5LbebQSibsSFFYqBjRtLeMMbLPzwS0EtwJTXSXpvhnYue7RvxX4xBrIkQCx58A3g6jFV6ZsqkfB1Ab2Fu6Og",
  "payload": ""
}
2025-02-09 21:08:37,405:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2219103505/473521759415 HTTP/1.1" 200 820
2025-02-09 21:08:37,405:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 09 Feb 2025 21:08:37 GMT
Content-Type: application/json
Content-Length: 820
Connection: keep-alive
Boulder-Requester: 2219103505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 4v_-BYbeem4XDLcq-SPI9emv4vajozdb1GUD7dnt_gmovpN-Cn0
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "secretbay.me"
  },
  "status": "pending",
  "expires": "2025-02-16T21:08:37Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2219103505/473521759415/E_FV7g",
      "status": "pending",
      "token": "KLrAYEIuVhPmsDwsZ_75afOIREZIgnKU7ift4HFPxkU"
    },
    {
      "type": "dns-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2219103505/473521759415/z7Wx_w",
      "status": "pending",
      "token": "KLrAYEIuVhPmsDwsZ_75afOIREZIgnKU7ift4HFPxkU"
    },
    {
      "type": "tls-alpn-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2219103505/473521759415/hDfckA",
      "status": "pending",
      "token": "KLrAYEIuVhPmsDwsZ_75afOIREZIgnKU7ift4HFPxkU"
    }
  ]
}
2025-02-09 21:08:37,406:DEBUG:acme.client:Storing nonce: 4v_-BYbeem4XDLcq-SPI9emv4vajozdb1GUD7dnt_gmovpN-Cn0
2025-02-09 21:08:37,406:INFO:certbot._internal.auth_handler:Performing the following challenges:
2025-02-09 21:08:37,407:INFO:certbot._internal.auth_handler:http-01 challenge for secretbay.me
2025-02-09 21:08:37,407:DEBUG:acme.standalone:Successfully bound to :80 using IPv6
2025-02-09 21:08:37,407:DEBUG:acme.standalone:Certbot wasn't able to bind to :80 using IPv4, this is often expected due to the dual stack nature of IPv6 socket implementations.
2025-02-09 21:08:37,409:DEBUG:acme.client:JWS payload:
b'{}'
2025-02-09 21:08:37,411:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall/2219103505/473521759415/E_FV7g:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjIxOTEwMzUwNSIsICJub25jZSI6ICI0dl8tQlliZWVtNFhETGNxLVNQSTllbXY0dmFqb3pkYjFHVUQ3ZG50X2dtb3ZwTi1DbjAiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLzIyMTkxMDM1MDUvNDczNTIxNzU5NDE1L0VfRlY3ZyJ9",
  "signature": "a8e9PZOvSiCsQHu4_DMjDZvM4CbLz4OXqmUo5OaRG5UuHyfA8WDoiQXJc1PrXI2zfvZ_Wfnw_R_w1lFVET0ZViqyqmfsxWKpKe9qJJnRAKSfY2Xe1JNnCaRaxiiz_FJ6-eKFPgsrX7v5ZZfZ6qj3oL5L7GjvJagjQLDpzJJw87lMMiUDPg-OSu3E3d9XX0BDB1xBiIqP08Iz6KX2TVWtcjIMmxz9069dL53SU0oG4GLE08l19oiqoLVULm6BV5DE62D2omJOlvNOEUIsHA6T7ybwg_c_umBqMw7kBDMuKBF2TPTNyMS4XcSXt4xG7ernUCCT9LpVZ5unQoFYcipXYg",
  "payload": "e30"
}
2025-02-09 21:08:37,574:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall/2219103505/473521759415/E_FV7g HTTP/1.1" 200 195
2025-02-09 21:08:37,575:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 09 Feb 2025 21:08:37 GMT
Content-Type: application/json
Content-Length: 195
Connection: keep-alive
Boulder-Requester: 2219103505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz/2219103505/473521759415>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall/2219103505/473521759415/E_FV7g
Replay-Nonce: VsUfN5aw3204nbaYBPtb-rhhp63AVu8C7h8J-G4GkgwV3AXCFk4
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2219103505/473521759415/E_FV7g",
  "status": "pending",
  "token": "KLrAYEIuVhPmsDwsZ_75afOIREZIgnKU7ift4HFPxkU"
}
2025-02-09 21:08:37,575:DEBUG:acme.client:Storing nonce: VsUfN5aw3204nbaYBPtb-rhhp63AVu8C7h8J-G4GkgwV3AXCFk4
2025-02-09 21:08:37,575:INFO:certbot._internal.auth_handler:Waiting for verification...
2025-02-09 21:08:38,576:DEBUG:acme.client:JWS payload:
b''
2025-02-09 21:08:38,578:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2219103505/473521759415:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjIxOTEwMzUwNSIsICJub25jZSI6ICJWc1VmTjVhdzMyMDRuYmFZQlB0Yi1yaGhwNjNBVnU4QzdoOEotRzRHa2d3VjNBWENGazQiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIyMTkxMDM1MDUvNDczNTIxNzU5NDE1In0",
  "signature": "F1lxTWZKu19zBEt3frk4Uj6asuy2p9tfXdFtX0CbRDDFChHRRiMygtR03efSbzExPiUVJvdyS2BlhLwpvvXuqm35b9ApLcMURKwDEeeAO7c7dM4uP8bBTOT9WHpRaZUF2W-3BK8Zz8u9Dd4XB8KcFJ-HVCrIInd3B2je-z0pCqSLAYgXRL9H4wbFyFu0nG9FOlBUPrioBoISV0jfM3424dZBdE1K0t516z2THdw3aXOkp2vA9rKVgIRPi3jEWWlkrA1UlG-dYKev9JAs-IERUAHGu9Gn2k2Zkt8IW2CcbExm4fYutoHHGIGEQqAK9cPBewEbq0ao3AKLpOpqtitAzw",
  "payload": ""
}
2025-02-09 21:08:38,742:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2219103505/473521759415 HTTP/1.1" 200 625
2025-02-09 21:08:38,743:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 09 Feb 2025 21:08:38 GMT
Content-Type: application/json
Content-Length: 625
Connection: keep-alive
Boulder-Requester: 2219103505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: VsUfN5aw11KdX8IBowwT2KDODLg3HfPHR0mtRBBalpl4nC0bQtU
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "secretbay.me"
  },
  "status": "invalid",
  "expires": "2025-02-16T21:08:37Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2219103505/473521759415/E_FV7g",
      "status": "invalid",
      "validated": "2025-02-09T21:08:37Z",
      "error": {
        "type": "urn:ietf:params:acme:error:dns",
        "detail": "no valid A records found for secretbay.me; no valid AAAA records found for secretbay.me",
        "status": 400
      },
      "token": "KLrAYEIuVhPmsDwsZ_75afOIREZIgnKU7ift4HFPxkU"
    }
  ]
}
2025-02-09 21:08:38,743:DEBUG:acme.client:Storing nonce: VsUfN5aw11KdX8IBowwT2KDODLg3HfPHR0mtRBBalpl4nC0bQtU
2025-02-09 21:08:38,743:INFO:certbot._internal.auth_handler:Challenge failed for domain secretbay.me
2025-02-09 21:08:38,744:INFO:certbot._internal.auth_handler:http-01 challenge for secretbay.me
2025-02-09 21:08:38,744:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: standalone). The Certificate Authority reported these problems:
  Domain: secretbay.me
  Type:   dns
  Detail: no valid A records found for secretbay.me; no valid AAAA records found for secretbay.me

Hint: The Certificate Authority failed to download the challenge files from the temporary standalone webserver started by Certbot on port 80. Ensure that the listed domains point to this machine and that it can accept inbound connections from the internet.

2025-02-09 21:08:38,746:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2025-02-09 21:08:38,746:DEBUG:certbot._internal.error_handler:Calling registered functions
2025-02-09 21:08:38,746:INFO:certbot._internal.auth_handler:Cleaning up challenges
2025-02-09 21:08:38,747:DEBUG:certbot._internal.plugins.standalone:Stopping server at :::80...
2025-02-09 21:08:38,910:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1873, in main
    return config.func(config, plugins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1579, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 142, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 518, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 429, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 497, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2025-02-09 21:08:38,915:ERROR:certbot._internal.log:Some challenges have failed.
